app/jinja_filters.py

```diff
@@ -709,8 +709,11 @@ def map_list_collector_config(
             else None
         )
 
+        iconVisuallyHiddenText = "Section complete" if icon == "check" else None
+
         row_item = {
             "iconType": icon,
+            "iconVisuallyHiddenText": iconVisuallyHiddenText,
```

Comment:
We don't use camel case for python vars, but not sure we need a var for this anyway, since it is only used once?

Reply:
Updated the var name 👍 We need this var, just in case render_icon is set to False, we don't want the "section complete" to be present, and if it is set to True we want it to be present, so I set the var up to control this

Reply:
But what I mean is it could just be `iconVisuallyHiddenText: "Section complete" if icon else None`? Since icon will always be either check or None?

Reply:
Updated 👍